src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java

```diff
@@ -99,7 +99,7 @@ public DiseaseAnnotation deprecateOrDeleteAnnotationAndNotes(Long id, Boolean th
 		if (deprecateAnnotation) {
 			if (!annotation.getObsolete()) {
 				annotation.setObsolete(true);
-				if (authenticatedPerson != null) {
+				if (authenticatedPerson.getOktaEmail() != null) {
```

Comment:
Can't just check for null authenticatedPerson as on bulk loads authenticatedPerson is a non-null object full of null fields, so updatedBy just gets set to null.

Reply:
Ahh I didn't think of this case.  I think it would be better to have it null vs an object with all null fields.

Reply:
Have changed this to `if (authenticatedPerson.getId() != null)` - don't know why I used OktaEmail field to check!

Reply:
We can change it so that authenticatedPerson is null? Or leave it?

Reply:
Not sure that I want to start tinkering with authenticatedPerson at this point (the details of the whole authenticatedPerson creation are still a little hazy to me) - maybe a separate PR